[Thread-4] WEBORB ERROR:18/12/2009 08:40:22:System.ArgumentException: The parameter 'address' cannot be an empty string.
Parameter name: address
   at System.Net.Mail.MailAddress..ctor(String address, String displayName, Encoding displayNameEncoding)
   at System.Net.Mail.MailAddress..ctor(String address, String displayName)
   at SDE.CamadaNuvem.NuvemModificacoes.EnviaEmail(String fromMailAddressData, String fromNameData, String fromPasswordData, String toMailAddressData, String toNameData, String subjectData, String messageData, String urlData) in C:\dev\sistemadaempresa\ambiente_desenvolvimento\marcos\lado_servidor\sde_marcos\SDE\code\CamadaNuvem\NuvemModificacoes_EnviaEmail.cs:line 35
[Thread-4] WEBORB ERROR:18/12/2009 11:14:09:Db4objects.Db4o.Ext.DatabaseFileLockedException: C:\dev\sistemadaempresa\ambiente_desenvolvimento\marcos\banco_de_dados\Banco\0.dbf ---> System.IO.IOException: The process cannot access the file 'C:\dev\sistemadaempresa\ambiente_desenvolvimento\marcos\banco_de_dados\Banco\0.dbf' because it is being used by another process.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   at Sharpen.IO.RandomAccessFile..ctor(String file, Boolean readOnly, Boolean lockFile)
   at Db4objects.Db4o.IO.RandomAccessFileFactory.NewRandomAccessFile(String path, Boolean readOnly, Boolean lockFile)
   --- End of inner exception stack trace ---
   at Db4objects.Db4o.IO.RandomAccessFileFactory.NewRandomAccessFile(String path, Boolean readOnly, Boolean lockFile)
   at Db4objects.Db4o.IO.FileStorage.FileBin..ctor(BinConfiguration config)
   at Db4objects.Db4o.IO.FileStorage.Open(BinConfiguration config)
   at Db4objects.Db4o.IO.StorageDecorator.Open(BinConfiguration config)
   at Db4objects.Db4o.IO.CachingStorage.Open(BinConfiguration config)
   at Db4objects.Db4o.Internal.IoAdaptedObjectContainer.OpenImpl()
   at Db4objects.Db4o.Internal.ObjectContainerBase._IRunnable_104.Run()
   at Db4objects.Db4o.Foundation.DynamicVariable.With(Object value, IRunnable block)
   at Db4objects.Db4o.Foundation.Environments.RunWith(IEnvironment environment, IRunnable runnable)
   at Db4objects.Db4o.Internal.ObjectContainerBase.WithEnvironment(IRunnable runnable)
   at Db4objects.Db4o.Internal.ObjectContainerBase.Open()
   at Db4objects.Db4o.Internal.IoAdaptedObjectContainer..ctor(IConfiguration config, String fileName)
   at Db4objects.Db4o.Internal.ObjectContainerFactory.OpenObjectContainer(IConfiguration config, String databaseFileName)
   at Db4objects.Db4o.Db4oFactory.OpenFile(IConfiguration config, String databaseFileName)
   at Db4objects.Db4o.Db4oFactory.OpenFile(String databaseFileName)
   at GerenteConectividadeBancoDados.get(Int32 idCorp, TipoBanco tipoBanco) in C:\dev\sistemadaempresa\ambiente_desenvolvimento\marcos\lado_servidor\sde_marcos\SDE\code\pattern\GerenteConectividadeBancoDados.cs:line 99
   at SDE.CamadaNuvem.SuperNuvem..ctor() in C:\dev\sistemadaempresa\ambiente_desenvolvimento\marcos\lado_servidor\sde_marcos\SDE\code\CamadaNuvem\SuperNuvem.cs:line 17
   at SDE.CamadaNuvem.NuvemNotificacoes..ctor()
[Thread-15] WEBORB ERROR:18/12/2009 15:14:16:An item with the same key has already been added.    at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at Weborb.Util.License.LicenseManager.GetInstance(Int32 productID)
   at Weborb.Util.Invocation.invoke(Object obj, MethodInfo method, Object[] arguments)
   at Weborb.Handler.ObjectHandler.invoke(String typeName, String function, Object[] arguments, RequestContext context)
[Thread-4] WEBORB ERROR:18/12/2009 15:22:55:An item with the same key has already been added.    at System.ThrowHelper.ThrowArgumentException(ExceptionResource resource)
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
   at Weborb.Util.License.LicenseManager.GetInstance(Int32 productID)
   at Weborb.Util.Invocation.invoke(Object obj, MethodInfo method, Object[] arguments)
   at Weborb.Handler.ObjectHandler.invoke(String typeName, String function, Object[] arguments, RequestContext context)
[Thread-4] WEBORB ERROR:18/12/2009 15:22:55:System.NullReferenceException: Object reference not set to an instance of an object.
   at System.Collections.Generic.Dictionary`2.Insert(TKey key, TValue value, Boolean add)
   at System.Collections.Generic.Dictionary`2.set_Item(TKey key, TValue value)
   at Weborb.Writer.ObjectWriter.write(Object obj, IProtocolFormatter writer)
   at b.a(Object A_0, IProtocolFormatter A_1)
   at Weborb.Writer.MessageWriter.writeObject(Object obj, IProtocolFormatter formatter)
   at Weborb.Writer.AMFBodyWriter.write(Object obj, IProtocolFormatter writer)
   at Weborb.Writer.MessageWriter.writeObject(Object obj, IProtocolFormatter formatter)
   at Weborb.Writer.AMFMessageWriter.write(Object obj, IProtocolFormatter writer)
   at Weborb.Writer.MessageWriter.writeObject(Object obj, IProtocolFormatter formatter)
   at Weborb.ORBHttpHandler.a(Request A_0, HttpRequest A_1, HttpResponse A_2, RequestContext A_3)
   at Weborb.ORBHttpHandler.a(HttpContext A_0, HttpRequest A_1, HttpResponse A_2)
